common/defs/blockchain_link.json

```diff
@@ -269,4 +283,4 @@
     "type": "blockbook",
     "url": ["https://sepolia1.trezor.io"]
   }
-}
+}
```

Comment:
missing trailing newline (editorcheck job is failing on this)

Reply:
beb153a026ec5550d0f931bbae21be71758333fc